tools/api-markdown-documenter/src/commands/test/lint-api-model.test.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import Path from "node:path";
+import { fileURLToPath } from "node:url";
+
+import { runCommand } from "@oclif/test";
+import { expect } from "chai";
+
+const dirname = Path.dirname(fileURLToPath(import.meta.url));
+
+// Mimic the behavior of running the command from the package root.
+const workingDirectory = Path.resolve(dirname, "..", "..", "..");
+
+describe("lint-api-model Command", () => {
```

Comment:
Do we have a simple api model we can use as a test case for "doesn't error unexpectedly on simple cases"?